docs/reference/search/request/post-filter.asciidoc

```diff
@@ -1,8 +1,9 @@
-[[request-body-search-post-filter]]
-==== Post filter
+[discrete]
+[[post-filter]]
+=== Post filter
 
-The `post_filter` is applied to the search `hits` at the very end of a search
-request,  after aggregations have already been calculated. Its purpose is
+The `post_filter` parameter is applied to the search `hits` at the very end of a search
```

Comment:
This is a bit awkward because the parameter isn't applied, the filter is. Maybe:

When you use the `post_filter` parameter to filter search results, the search hits are filtered _after_ the aggregations are calculated. A post filter has no impact on the aggregation results.